neuro_data/static_images/data_schemas.py

```diff
@@ -163,10 +165,10 @@ def fill(self, scan_key):
                     skip_duplicates=True)
         self.insert([{'image_id': iid, 'image_class': ic, 'tier': 'validation'} for
                      iid, ic in zip(image_ids[num_oracles: num_oracles + num_validation],
-                                    image_classes[num_oracles: num_oracles + num_validation])])
+                                    image_classes[num_oracles: num_oracles + num_validation])], skip_duplicates=True)
```

Comment:
I think I left out the skip_duplicates so people won't call it twice with the same ImageNet collection. If you are sure it will be ok for people to cal it many times with the same collection (or scan that was shown the same ImageNet collection), it should be ok

Reply:
My intention was to allow people to call it multiple times, since either way if they call it again nothing will change assuming the ImageNet collection is the same. I added this cause I think for the color MEI we had some shared images but different collections.